src/opnsense/www/js/widgets/Gateways.js

```diff
@@ -65,15 +65,15 @@ export default class Gateways extends BaseTableWidget {
             gateways: {
                 title: this.translations.title,
                 type: 'select_multiple',
-                options: this.cachedGateways.map((name) => {
+                options: this.cachedGateways.map(({ name, uuid }) => {
                     return {
-                        value: name,
+                        value: uuid,
```

Comment:
I suppose this implicates that current configurations are now invalid? or does `name == uuid` the the old vs new api call?

If invalid, good thing we now have https://github.com/opnsense/core/commit/b52055af0b2ae2851905c132b9597dbc7d07c28f + https://github.com/opnsense/core/commit/7373985f3b2b0344c1e2596bdbbb5b0870cadb57

Reply:
Yeah the configuration will be invalid since it will be mapped to uuid now. If its okay to push this as new default with your commits you made that ensure a default, we can leave it like this.

If not, I can revert this back to name but having uuid here would be better in the long run I think.